docs/azure/vscodeforweb.md

```diff
@@ -0,0 +1,102 @@
+---
+Order: 5
+Area: azure
+TOCTitle: VS Code for the Web
+PageTitle: Develop and Deploy to Azure in VS Code for the Web
+ContentId:
+MetaDescription: VS Code for the Web
+DateApproved: 05/12/2025
+---
+
+
+VS Code for the Web is a zero-install and browser-based version of Visual Studio Code. The `/azure` environment—accessible via <https://insiders.vscode.dev/azure> is a dedicated space for Azure development, allowing you to run, debug, and deploy applications to Azure in seconds.
```

Comment:
`/azure` environment sounds like a technical term. Is there a more user-friendly name for this?